engine/schema/src/com/cloud/domain/DomainVO.java

```diff
@@ -135,6 +139,10 @@ public long getAccountId() {
         return accountId;
     }
 
+    public void setAccountId(long accountId) {
+        this. accountId= accountId;
```

Comment:
@ustcweizhou can you please remove the blank space between 'this.' and 'accountId'?